pkg/github/github_app.go

```diff
@@ -25,17 +25,6 @@ type response struct {
 	AppName      string
 }
 
-func (gh *GithubApp) isGithubAppEnabled() (bool, error) {
```

Comment:
Revert this, please - this is used in a downstream dependency.

Reply:
How can that be? A downstream dependency uses a non exported function?

Reply:
ah, yeah, I'm wrong, ignore me.